.env.template

```diff
@@ -14,3 +14,11 @@ TWOOPSTRACKER_ALLOWED_HOSTS=
 TWOOPSTRACKER_DEBUG=False
 TWOOPSTRACKER_CORS_ALLOWED_ORIGINS=
 TWOOPSTRACKER_SENTRY_DSN=
+ACCOUNT_CONFIRM_EMAIL_ON_GET=False
```

Comment:
We can use the `TWOOPSTRACKER_` naming just for consistency.

Reply:
@esirK these are package related settings

Reply:
I see. 
Although I meant something like 
```
TWOOPSTRACKER_ACCOUNT_CONFIRM_EMAIL_ON_GET=False
```
in the env file and then in the settings file we can have 

```
ACCOUNT_CONFIRM_EMAIL_ON_GET=env.bool("TWOOPSTRACKER_ACCOUNT_CONFIRM_EMAIL_ON_GET", False)
```
although I don't think this is a big deal.